sklearn/linear_model/_logistic.py

```diff
@@ -827,6 +821,11 @@ class LogisticRegression(LinearClassifierMixin, SparseCoefMixin, BaseEstimator):
     the L2 penalty. The Elastic-Net regularization is only supported by the
     'saga' solver.
 
+    For :term:`multiclass` problems, only 'newton-cg', 'sag', 'saga' and 'lbfgs'
```

Comment:
@jeremiedbb Since we are in the long summary and we have an alternative to use OvR, I would advocate from now on to use `OneVsRestClassifier`. It looks legit to me.

Reply:
alright then :)